### PR TITLE
MANTA-5426 madtom also affected by TRITON-2173

### PR DESCRIPTION
**Pull request by bahamat**

### CODE REVIEW

**Comment by bahamat** [comment]
DO NOT MERGE

This is a ***DRAFT***

Requires node-checker hash to be updated after joyent/node-checker#45 is merged.

**Review by twhiteman — APPROVED** [review]

**Pull request merged**
